tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/FilterExecParser.scala

```diff
@@ -32,14 +32,15 @@ case class FilterExecParser(
     val duration = None
     val exprString = node.desc.replaceFirst("Filter ", "")
     val expressions = SQLPlanParser.parseFilterExpressions(exprString)
-    val isAllExprsSupported = expressions.forall(expr => checker.isExprSupported(expr))
+    val notSupportedExprs = expressions.filter(expr => !checker.isExprSupported(expr))
```

Comment:
nit: idiomatic filterNot
```suggestion
    val notSupportedExprs = expressions.filterNot(expr => checker.isExprSupported(expr))
```

Reply:
done. PTAL